### DISCUSSION TITLE
VC event google calendar feed?

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1194] [New Ideas]
It might be cool to offer the ability for people to add the VC event calendar as a custom google calendar, or via Apple Calendar.

Looking into it, I thought maybe CraftCMS calendar plugin provides this out of the box. It seems like they do with the Pro version. 

<img width="509" alt="Screenshot 2024-06-26 at 10 05 37 AM" src="https://github.com/Virtual-Coffee/virtualcoffee.io/assets/2068912/b5fbcf37-1166-449e-830c-8a2d90b5802c">

https://plugins.craftcms.com/calendar

Are y'all on the Pro version or the Lite version?

**Comment by BekahHW** [comment]
@danieltott?

**Comment by danieltott** [comment]
Yes, that'd be cool. The problem is we have direct links to zoom in the calendar, which is fine because it posts to slack. If we put the calendar online, we'd have to handle a lot more authentication issues (even if we only shared it with members).

**Reply by ebanner** [reply]
Hm, I'm not understanding why we would need to handle a lot more authentication issues if we only shared it with members?

**Comment by danieltott** [comment]
Because people can do things like share calendar events, invite people to events, etc etc. This is the same problem if we start emailing people - once a thing is out there, it's a lot easer for people to share it to others without thinking.

If they're clicking on a link in slack, we know for sure they're logged in to our slack. If they click from somewhere else, we have no way of knowing ahead of time.

I'm not saying it's a bad idea, just a hard one.

Also - the Calendar craft plugin creates ICS for each event type, which is really annoying. So to use the default without additions, users would have to subscribe to 3 calendars if they wanted Coffees, Lunch & Learns, and Front End Fridays for instance.

**Reply by ebanner** [reply]
Oh interesting! Thanks for sharing.

Yeah ok, given that, the options seem to be 

- Create some zoom org (?) where only people part of the org can join the zoom calls
- Put a password on the zoom calls that we rotate 

Though the simplest solution seems to be just removing the zoom links from the calendar events (maybe just link to the slack)
